packages/frint-cli/commands/new.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable no-multi-spaces */
+
+const mkdirp = require('mkdirp');
+const request = require('request');
+const tar = require('tar');
+
+const createApp = require('frint').createApp;
+
+const descriptionText = `
+Usage:
+
+  $ frint new
+  $ frint new <directory>
+  $ frint new <directory> --example <example>
+
+Example:
+
+  $ frint new myapp --example kitchensink
+  $ frint new myapp --example frint-vue/tree/master/examples/basic
+
+You can find a list of all available official examples here:
+https://github.com/Travix-International/frint/tree/master/examples
+`.trim();
+
+const invalidExampleArgText = `
+Invalid <example> value. Must be in one of the following formats:
+
+  * <name>
+  * <organization>/<repository>/tree/<branch>/**/<name>
+`.trim();
+
+module.exports = createApp({
+  name: 'new',
+  providers: [
+    {
+      name: 'summary',
+      useValue: 'Scaffolds a new Frint app in specified directory',
+    },
+    {
+      name: 'description',
+      useValue: descriptionText,
+    },
+    {
+      name: 'execute',
+      useFactory: function useFactory(deps) {
+        return function execute() {
+          // The <example> param has two shapes:
+          // * <name> - example name from the official Frint GitHub repository
+          // * <organization>/<repo>/tree/<branch>/** - full GitHub path to an arbitrary example
+          const example = deps.params.example || 'counter';
```

Comment:
not really your doing, it was me who set the default example value deep inside before.

but it is good practice to define things like this at the top of the file as a constance, like:

```js
const DEFAULT_EXAMPLE = 'counter';
```

until we remove the `init` command, both `new` and `init` can share this same constant.